packages/cli/src/services/orchestration/main/MultiMainSetup.ee.ts

```diff
@@ -69,24 +80,44 @@ export class MultiMainInstancePublisher extends SingleMainInstancePublisher {
 		} else {
 			this.logger.debug(`Leader is other instance "${leaderId}"`);
 
-			this.leaderId = leaderId;
+			config.set('multiMainSetup.instanceType', 'follower');
 		}
 	}
 
 	private async tryBecomeLeader() {
-		if (this.isLeader || !this.redisPublisher.redisClient) return;
+		if (
+			config.getEnv('multiMainSetup.instanceType') === 'leader' ||
+			!this.redisPublisher.redisClient
+		) {
+			return;
+		}
 
 		// this can only succeed if leadership is currently vacant
 		const keySetSuccessfully = await this.redisPublisher.setIfNotExists(this.leaderKey, this.id);
 
 		if (keySetSuccessfully) {
 			this.logger.debug(`Leader is now this instance "${this.id}"`);
 
-			this.leaderId = this.id;
-
-			this.emit('leadershipChange', this.id);
+			config.set('multiMainSetup.instanceType', 'leader');
 
 			await this.redisPublisher.setExpiration(this.leaderKey, this.leaderKeyTtl);
+
+			this.emit('leadershipChange', this.id);
+		} else {
+			config.set('multiMainSetup.instanceType', 'follower');
 		}
 	}
+
+	async broadcastWorkflowActiveStateChanged(payload: {
+		workflowId: string;
+		oldState: boolean;
+		newState: boolean;
+	}) {
+		if (!this.sanityCheck()) return;
+
+		await this.redisPublisher.publishToCommandChannel({
+			command: 'workflowActiveStateChanged',
```

Comment:
```suggestion
			command: 'reloadWorkflowActiveState',
```

Based on the conversation yesterday, commands should indicate imperative actions.

Reply:
From what I recall, what we decided yesterday was that these were not commands but rather events, so it'd be the others (and methods, etc.) that need renaming. 

In any case, I tend to think we should avoid a single one being inconsistent with the others - ideally we could rename all of this implementation in a standalone PR.